packages/block-library/src/search/edit.js

```diff
@@ -223,11 +224,13 @@ export default function SearchEdit( {
 		return (
 			<>
 				{ buttonUseIcon && (
-					<Button
-						icon={ search }
+					<button
+						type="button"
```

Comment:
the old component button had `type="button"` but the frontend has `type="submit"` so it's probably best to use submit here